SmartPlaces.Facilities/products/Topology/src/README.md

```diff
@@ -0,0 +1,51 @@
+﻿# Topology
+
+## Overview
+The Topology project is an example of how to pull the topology from a Mapped Instance and insert a matching topology into an instance of Azure Digital Twins
+
+## Prerequisites
+
+1. An instance of Mapped with a populated graph of objects
+2. A Personal Access Token (PAT) for the Mapped instance
+3. An Azure Digital Twins Instance
+4. An instance of Redis Cache running in the cloud
+
+## Data Flow
+
+1. Topology is pulled from Mapped instance
+2. Mapped instances are converted to the appropriate target ontology
+3. Graph is loaded into Azure Digital Twins
+4. A mapping from the Mapped mappingKey to the ADT Twin ID is added to the Redis Cache for each twin
+
+## Configuring
+
+### Environment Variables
+
+The following environment variables need to be defined in order to run this code:
+
+| Environment Variable Name | Value Source | Example |
+| --- | --- | --- |
+| KeyVaultEndpoint | The Uri to the KeyVault instance where needed secrets are stored | https://<yourkeyvaultname>.vault.azure.net/ |
+| AzureDigitalTwinsEndpoint | The Uri of the ADT Instance to update | https://<youradtname>.api.<yourregion>.digitaltwins.azure.net |
+| RedisCacheConnectionString | The endpoint location of the local Redis cluster in the cloud that store the mapping from the MappingKey to the ADT Twin ID | <yourcachename>.redis.cache.windows.net,abortConnect=false,ssl=true,allowAdmin=true,password=<yourpassword> |
```

Comment:
Do we really assign a password as an environment variable? That isn't very secure

Reply:
Referring to the Redis Cache? That connection string does not include the password...

Reply:
`password=<yourpassword>` in the example indicates otherwise

Reply:
It was in the wrong section. This has been moved to the KeyVault. Updated docs